minimesos/src/test/java/com/containersol/minimesos/main/CommandUpTest.java

```diff
@@ -1,141 +1,86 @@
+    @Test(expected = MinimesosException.class)
+    public void testExecute_invalidMinimesosFile() throws IOException {
+        commandUp.setClusterConfigPath("src/test/resources/configFiles/invalid-minimesosFile");
+        commandUp.execute();
     }
 
     @Test
-    public void testUsingExposedPortsFromConfigFileFalse() {
-        ClusterConfig clusterConfig = new ClusterConfig();
-        clusterConfig.setExposePorts(false);
-
-        CommandUp commandUp = new CommandUp();
-        commandUp.updateWithParameters(clusterConfig);
+    public void testBasicClusterConfig() throws IOException {
+        commandUp.setClusterConfigPath("src/test/resources/clusterconfig/basic.groovy");
+        commandUp.execute();
 
-        assertFalse("Exposed port from configuration is expected to remain", clusterConfig.isExposePorts());
+        verify(mesosCluster).start();
     }
 
     @Test
-    public void testOverwritingExposedPortsFromCommand() {
-        ClusterConfig clusterConfig = new ClusterConfig();
-        clusterConfig.setExposePorts(true);
-
-        CommandUp commandUp = new CommandUp();
-        commandUp.setExposedHostPorts(false);
-        commandUp.updateWithParameters(clusterConfig);
+    public void testExecute_setTimeout() {
+        commandUp.setClusterConfigPath("src/test/resources/configFiles/minimal-minimesosFile");
+        commandUp.setTimeout(30);
+        commandUp.execute();
 
-        assertFalse("Exposed port should be changed through command parameters", clusterConfig.isExposePorts());
+        assertEquals(30, capturedClusterConfig.getValue().getTimeout());
     }
 
     @Test
-    public void testMinimalConfig() throws InterruptedException {
-        CommandUp commandUp = new CommandUp();
+    public void testExecute_exposedPorts() {
         commandUp.setClusterConfigPath("src/test/resources/configFiles/minimal-minimesosFile");
+        commandUp.setExposedHostPorts(true);
+        commandUp.execute();
 
-        ClusterArchitecture clusterArchitecture = commandUp.getClusterArchitecture();
-
-        assertEquals(1, clusterArchitecture.getClusterContainers().getContainers().stream().filter(c -> c instanceof ZooKeeper).count());
-        assertEquals(1, clusterArchitecture.getClusterContainers().getContainers().stream().filter(c -> c instanceof MesosMaster).count());
-        assertEquals(1, clusterArchitecture.getClusterContainers().getContainers().stream().filter(c -> c instanceof MesosAgent).count());
-        assertEquals(3, clusterArchitecture.getClusterContainers().getContainers().size());
+        assertTrue("Exposed port from configuration is expected to remain", capturedClusterConfig.getValue().isExposePorts());
     }
 
     @Test
-    public void testMarathonAppConfig() throws InterruptedException {
-        CommandUp commandUp = new CommandUp();
-        commandUp.setClusterConfigPath("src/test/resources/configFiles/marathonAppConfig-minimesosFile");
-
+    public void testExecute_() {
```

Comment:
Tests `testExecute_()` and `testExecute_setTimeout()` are identical. Please, resolve